assets/javascripts/discourse/components/modal/activity-pub-actor-reject.hbs

```diff
@@ -0,0 +1,24 @@
+<DModal
```

Comment:
If this is just a confirmation dialog, you can also use the dialog service. Very similar end result, but less code.

Reply:
We'd need to also change all the other modals in ActivityPubFollowBtn mapping to do this: https://github.com/discourse/discourse-activity-pub/pull/123/files#diff-9a9f8850d2acb2e99c904d92233b145404da4d8487cf051f347c145550fead52R16

Reply:
I thought this was an isolated modal. In that case, might not be worth switching over.